packages/compass-connections-navigation/src/constants.tsx

```diff
@@ -9,16 +9,23 @@ export const ROW_HEIGHT = spacing[5];
 // export const COLLETIONS_MARGIN_BOTTOM = spacing[1];
 
 export type Actions =
```

Comment:
i agree that it will be good for organisation but I won't be able to reap the TS benefits from it because we mostly depend on the SidebarActionableItem in the sidebar to derive the necessary parameters (namespace, connectionId) to be passed to the other methods. So I will probably need some TS magic to bind the item type to the enums?